azure-storage-cpp-lite/src/blob/blob_client.cpp

```diff
@@ -65,6 +65,7 @@ storage_outcome<chunk_property> blob_client::get_chunk_to_stream_sync(const std:
         property.etag = http->get_header(constants::header_etag);
         property.totalSize = get_length_from_content_range(http->get_header(constants::header_content_range));
         std::istringstream(http->get_header(constants::header_content_length)) >> property.size;
+        property.last_modified = curl_getdate(http->get_header(constants::header_last_modified).c_str(), NULL);
```

Comment:
BTW, what does curl_getdata returns in case of NULL? Should we provide 1970 as a default value for last modified header?